app/controllers/posts_controller.rb

```diff
@@ -22,6 +22,16 @@ def create
     end
   end
 
+  def destroy
+    @post = Post.find(params[:id])
+
+    if @post.destroy
+      redirect_to user_posts_path(current_user), notice: t('.deleted')
+    else
+      redirect_to user_posts_path(current_user), alert: t('.not_deleted')
+    end
+  end
+
   private
 
   def post_params
```

Comment:
- [ ] As required for the project and to correctly setup the user authorization using [CanCanCan](https://github.com/CanCanCommunity/cancancan), kindly make sure you use **CanCanCan** for Post authorization in `Post Controller`. i.e.
```ruby
  class PostsController < ApplicationController
    load_and_authorize_resource

    ...
  end
```

Reply:
Hi @tresorsawasawa, Thanks for your review. We don't add it because we do not use any helper, we thought it was not necessary. But now we will add it.